packages/middleware-stack/src/index.spec.ts

```diff
@@ -33,22 +39,22 @@ describe('MiddlewareStack', () => {
         const stack = new MiddlewareStack<input, output>();
 
         const middleware = shuffle([
-            [concatMiddleware.bind(null, 'second')],
```

Comment:
What error was TypeScript throwing here? Can `undefined` still be used, but not `null`?

Reply:
I opened an issue #154, basically typescript will assume the middleware is `FinalizeMiddleware` even with `.bind()`(which it didn't before), and it complained about interface not compatible(`Middleware`->`FinalizeMiddleware`, `HandlerArguments`->`FinalizeHandlerArguments`). So I removed `.bind()` and used middleware directly.

Reply:
Ah interesting. I wonder if that'll be fixed when they support generics with bind/call/apply.